wurst/systems/boards/ScoreBoard.wurst

```diff
@@ -238,6 +238,9 @@ init
                     for member in tribe.getMembers()
                         scoreBoardEntries.get("Level")
                         .updateEntryIcon(member.getName(), member.getTrollIcon())
+                doPeriodically(1) (CallbackPeriodic cb) ->
+                    let title = getGameTimersBoardTitle()
```

Comment:
There's no point in having this local. `socreBoard.board.setTitle(getGameTimersBoardTitle())` is succinct enough I think